apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/LandNormalisert.kt

```diff
@@ -0,0 +1,287 @@
+package no.nav.etterlatte.trygdetid
+
+enum class LandNormalisert(val isoCode: String, val beskrivelse: String) {
+    // Sorted by ISO 3166-1 alpha-3 code:
+    ARUBA("ABW", "Aruba"),
+    AFGHANISTAN("AFG", "Afghanistan"),
+    ANGOLA("AGO", "Angola"),
+    ANGUILLA("AIA", "Anguilla"),
+    ALAND("ALA", "Åland"),
+    ALBANIA("ALB", "Albania"),
+    ANDORRA("AND", "Andorra"),
+    NEDERLANDSKE_ANTILLER("ANT", "De nederlandske Antiller"),
+    FORENTE_ARABISKE_EMIRATER("ARE", "De forente arabiske emirater"),
+    ARGENTINA("ARG", "Argentina"),
+    ARMENIA("ARM", "Armenia"),
+    AMERIKANSK_SAMOA("ASM", "Amerikansk Samoa"),
+    ANTARKTIS("ATA", "Antarktis"),
+    DE_FRANSKE_TERRITORIENE_I_SOR("ATF", "De franske territoriene i sør"),
+    ANTIGUA_OG_BARBUDA("ATG", "Antigua og Barbuda"),
+    AUSTRALIA("AUS", "Australia"),
+    OSTERRIKE("AUT", "Østerrike"),
+    AZERBAJDZJAN("AZE", "Azerbajdzjan"),
+    BURUNDI("BDI", "Burundi"),
+    BELGIA("BEL", "Belgia"),
+    BENIN("BEN", "Benin"),
+
+    // a.k.a. Karibisk Nederland eller BES-øyene
+    BONAIRE_SINT_EUSTATIUS_OG_SABA("BES", "Bonaire, Sint Eustatius og Saba"),
+    BURKINA_FASO("BFA", "Burkina Faso"),
+    BANGLADESH("BGD", "Bangladesh"),
+    BULGARIA("BGR", "Bulgaria"),
+    BAHRAIN("BHR", "Bahrain"),
+    BAHAMAS("BHS", "Bahamas"),
+    BOSNIA_HERCEGOVINA("BIH", "Bosnia-Hercegovina"),
+    SAINT_BARTHELEMY("BLM", "Saint Barthelemy"),
+    HVITERUSSLAND("BLR", "Hviterussland"),
```

Comment:
```suggestion
    BELARUS("BLR", "Belarus"),
```
https://no.wikipedia.org/wiki/Belarus :D

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1489/commits/d314f91ad1b297f419658137ba08a17115647bc8

Reply:
Nice catch